kyuubi-server/src/main/scala/org/apache/kyuubi/parser/plan/command/ShowSessions.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.parser.plan.command
+
+import scala.collection.mutable.ListBuffer
+
+import org.apache.kyuubi.operation.IterableFetchIterator
+import org.apache.kyuubi.parser.schema.{Column, Row, Schema}
+import org.apache.kyuubi.parser.types.StringType
+import org.apache.kyuubi.session.KyuubiSession
+
+/**
+ * A runnable node for showing kyuubi sessions.
+ *
+ * The syntax of using this command in SQL is:
+ * {{{
+ *   SHOW KYUUBI_SESSIONS;
+ * }}}
+ */
+case class ShowSessions() extends RunnableCommand {
+
+  override def run(kyuubiSession: KyuubiSession): Unit = {
+    val rows = kyuubiSession.sessionManager.allSessions().map { session =>
```

Comment:
So this cmd list all sessions, is it safe to do such thing from an end user? @pan3793

Reply:
```
org.apache.kyuubi.server.api.v1.SessionsResource#sessions
```

Reply:
The REST API is in experimental phase, not delared GA

Reply:
List all sessions is an insecure operation, we need to have a mechanism to prevent this insecure situation from happening, either the person who does this operation is admin, or only list the sessions that the user has permission to, it seems that there is no good mechanism now? This command can be set up as a experimental at this phase.

Reply:
let's remove this command from this PR

Reply:
ok

Reply:
agree with you, will add desc session command